Tests/SpeziAccountTests/AccountNotificationsTests.swift

```diff
@@ -49,64 +49,66 @@ private final actor TestStandard: Standard, AccountNotifyConstraint {
     }
 }
 
-
-final class AccountNotificationsTests: XCTestCase {
-    @MainActor
-    func testAccountNotifications() async throws {
-        let disassociatedExpectation = XCTestExpectation(description: "disassociated")
-        let deleteExpectation = XCTestExpectation(description: "delete")
-
-        let notifications = AccountNotifications()
-        let standard = TestStandard()
-        let provider = TestProvider(disassociatedExpectation: disassociatedExpectation, deleteExpectation: deleteExpectation)
-        withDependencyResolution(standard: standard) {
-            notifications
-            ExternalAccountStorage(provider)
-        }
-
-        let stream = notifications.events
-
-        let details: AccountDetails = .mock()
-
-        try await notifications.reportEvent(.deletingAccount("account1"))
-        try await notifications.reportEvent(.associatedAccount(details))
-        try await notifications.reportEvent(.disassociatingAccount(details))
-
-        var iterator = stream.makeAsyncIterator()
-
-        let element0 = await iterator.next()
-        let element1 = await iterator.next()
-        let element2 = await iterator.next()
-
-        func assertEvents(_ event0: AccountNotifications.Event?, _ event1: AccountNotifications.Event?, _ event2: AccountNotifications.Event?) {
-            if case let .deletingAccount(id) = element0 {
-                XCTAssertEqual(id, "account1")
-            } else {
-                XCTFail("Unexpected first element \(String(describing: element0))")
+// swiftlint:disable closure_body_length
```

Comment:
I would propose to add `// swiftlint:disable:this closure_body_length` comments to the respective lines where this warning is suppressed. This makes it a bit more clear where the warning is raised 👍